src/documentation/verifier.cpp

```diff
@@ -271,6 +273,57 @@ std::string DirectoryVerifier::type() const {
     return "Directory";
 }
 
+TestResult DateTimeVerifier::operator()(const ghoul::Dictionary& dict,
+    const std::string& key) const
```

Comment:
Nitpicky:  the second line should be aligned with the opening `(` of the parameter list on the line above